src/lib/messaging/channels/wechat/qr.ts

```
@@ -104,6 +116,115 @@ export class WechatQrError extends Error {
+          } catch {
+            // Preserve the bounded-response error.
+          }
+          throw new WechatQrError("parse", `WeChat QR ${label} response exceeded the size limit`);
+        }
+        chunks.push(value);
+      }
+    } finally {
+      try {
+        reader.releaseLock();
+      } catch (err) {
+        if (!signal.aborted) throw err;
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)